include/openspace/scene/scenegraphnode.h

```diff
@@ -170,15 +172,17 @@ class SceneGraphNode : public properties::PropertyOwner {
     std::vector<std::string> _onRecedeAction;
     std::vector<std::string> _onExitAction;
 
+    ghoul::mm_unique_ptr<Renderable> _renderable;
+
```

Comment:
Why moving it further up?

Reply:
Ah, just to group the gui properties. It bugged me that `_guiHidden` was not placed together with the other ones

But I'm fine with undoing this change

Reply:
Nah.. i was just curious